cypress/support/elements/clue/ChatPanel.js

```diff
@@ -25,7 +25,7 @@ class ChatPanel{
       return cy.get('[data-testid=comment-card]');
     }
     getCommentTileTypeIcon() {
-      return cy.get('[data-testid=chat-thread-tile-type]').find('svg');
+      return cy.get('[data-testid=chat-thread-tile-type]');
```

Comment:
This test is trying to find a SVG which has attribute` data-testid = chat-thread-tile-type` 
Using find was searching the child descendents of the SVG and it wasn't passing the test, so I took it out.